components/email-mgt/org.wso2.carbon.email.mgt/src/main/java/org/wso2/carbon/email/mgt/internal/TenantManagementListener.java

```diff
@@ -20,16 +20,11 @@
 
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
-import org.wso2.carbon.email.mgt.EmailTemplateManager;
-import org.wso2.carbon.email.mgt.EmailTemplateManagerImpl;
-import org.wso2.carbon.email.mgt.exceptions.I18nEmailMgtException;
-import org.wso2.carbon.identity.governance.exceptions.notiification.NotificationTemplateManagerException;
-import org.wso2.carbon.identity.governance.service.notification.NotificationChannels;
-import org.wso2.carbon.identity.governance.service.notification.NotificationTemplateManager;
 import org.wso2.carbon.stratos.common.beans.TenantInfoBean;
 import org.wso2.carbon.stratos.common.exception.StratosException;
 import org.wso2.carbon.stratos.common.listeners.TenantMgtListener;
 
+@Deprecated
 public class TenantManagementListener implements TenantMgtListener {
```

Comment:
Lets remove the full class as its not required anymore.

Reply:
Also lets avoid registering this as a tenant mgt listener in the I18nMgtServiceComponent

Reply:
Addressed with https://github.com/wso2-extensions/identity-event-handler-notification/pull/258/commits/2b9424930522b1efcfb51478d134f925b6393352